core/src/main/java/bio/terra/pearl/core/model/dashboard/AlertType.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.pearl.core.model.dashboard;
+
+public enum AlertType {
+    primary,
```

Comment:
Just realized while reviewing PR 663 that these enum elements should be capitalized.